aspnetcore/security/data-protection/configuration/overview.md

```diff
@@ -105,7 +105,7 @@ public void ConfigureServices(IServiceCollection services)
 {
     services.AddDataProtection()
         .PersistKeysToFileSystem(new DirectoryInfo(@"\\server\share\directory\"))
-        .ProtectKeysWithCertificate("thumbprint");
+        .ProtectKeysWithCertificate($CREDENTIAL_PLACEHOLDER$);
```

Comment:
You need to replace all occurrences of password in this document with a method secure method call.

Reply:
> .ProtectKeysWithCertificate("thumbprint");

@blowdart  does `"thumbprint"` need to be replaced with `GetThumbprint()` ?

Reply:
This is removing thumbprint. Umm, it depends, If it's a thumbprint in config then yes, or you pass a cert directly